src/app/shared/user.service.ts

```diff
@@ -7,15 +7,19 @@ import { HttpClient, HttpHeaders } from "@angular/common/http";
 export class UserService {
```

Comment:
Why the form builder and form model are defined inside the Service. Should it be in the User Component?

Reply:
changes done